2024-09-02 05:15:04.585 INFO page_ 1:conftest.py:34: new page opened: [<Page url='about:blank'>]
2024-09-02 05:15:04.596 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.address`.
2024-09-02 05:15:04.598 DEBUG faker.factory:factory.py:95: Provider `faker.providers.address` has been localized to `ru_RU`.
2024-09-02 05:15:04.613 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.automotive`.
2024-09-02 05:15:04.615 DEBUG faker.factory:factory.py:95: Provider `faker.providers.automotive` has been localized to `ru_RU`.
2024-09-02 05:15:04.617 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.bank`.
2024-09-02 05:15:04.619 DEBUG faker.factory:factory.py:95: Provider `faker.providers.bank` has been localized to `ru_RU`.
2024-09-02 05:15:04.625 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.barcode`.
2024-09-02 05:15:04.626 DEBUG faker.factory:factory.py:86: Specified locale `ru_RU` is not available for provider `faker.providers.barcode`. Locale reset to `en_US` for this provider.
2024-09-02 05:15:04.626 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.color`.
2024-09-02 05:15:04.627 DEBUG faker.factory:factory.py:95: Provider `faker.providers.color` has been localized to `ru_RU`.
2024-09-02 05:15:04.628 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.company`.
2024-09-02 05:15:04.630 DEBUG faker.factory:factory.py:95: Provider `faker.providers.company` has been localized to `ru_RU`.
2024-09-02 05:15:04.637 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.credit_card`.
2024-09-02 05:15:04.637 DEBUG faker.factory:factory.py:95: Provider `faker.providers.credit_card` has been localized to `ru_RU`.
2024-09-02 05:15:04.649 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.currency`.
2024-09-02 05:15:04.650 DEBUG faker.factory:factory.py:95: Provider `faker.providers.currency` has been localized to `ru_RU`.
2024-09-02 05:15:04.655 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.date_time`.
2024-09-02 05:15:04.656 DEBUG faker.factory:factory.py:95: Provider `faker.providers.date_time` has been localized to `ru_RU`.
2024-09-02 05:15:04.668 DEBUG faker.factory:factory.py:106: Provider `faker.providers.emoji` does not feature localization. Specified locale `ru_RU` is not utilized for this provider.
2024-09-02 05:15:04.668 DEBUG faker.factory:factory.py:106: Provider `faker.providers.file` does not feature localization. Specified locale `ru_RU` is not utilized for this provider.
2024-09-02 05:15:04.668 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.geo`.
2024-09-02 05:15:04.669 DEBUG faker.factory:factory.py:86: Specified locale `ru_RU` is not available for provider `faker.providers.geo`. Locale reset to `en_US` for this provider.
2024-09-02 05:15:04.669 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.internet`.
2024-09-02 05:15:04.670 DEBUG faker.factory:factory.py:95: Provider `faker.providers.internet` has been localized to `ru_RU`.
2024-09-02 05:15:04.673 DEBUG faker.factory:factory.py:106: Provider `faker.providers.isbn` does not feature localization. Specified locale `ru_RU` is not utilized for this provider.
2024-09-02 05:15:04.674 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.job`.
2024-09-02 05:15:04.675 DEBUG faker.factory:factory.py:95: Provider `faker.providers.job` has been localized to `ru_RU`.
2024-09-02 05:15:04.680 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.lorem`.
2024-09-02 05:15:04.681 DEBUG faker.factory:factory.py:95: Provider `faker.providers.lorem` has been localized to `ru_RU`.
2024-09-02 05:15:04.685 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.misc`.
2024-09-02 05:15:04.685 DEBUG faker.factory:factory.py:86: Specified locale `ru_RU` is not available for provider `faker.providers.misc`. Locale reset to `en_US` for this provider.
2024-09-02 05:15:04.685 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.passport`.
2024-09-02 05:15:04.686 DEBUG faker.factory:factory.py:86: Specified locale `ru_RU` is not available for provider `faker.providers.passport`. Locale reset to `en_US` for this provider.
2024-09-02 05:15:04.686 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.person`.
2024-09-02 05:15:04.689 DEBUG faker.factory:factory.py:95: Provider `faker.providers.person` has been localized to `ru_RU`.
2024-09-02 05:15:04.689 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.phone_number`.
2024-09-02 05:15:04.691 DEBUG faker.factory:factory.py:95: Provider `faker.providers.phone_number` has been localized to `ru_RU`.
2024-09-02 05:15:04.693 DEBUG faker.factory:factory.py:106: Provider `faker.providers.profile` does not feature localization. Specified locale `ru_RU` is not utilized for this provider.
2024-09-02 05:15:04.693 DEBUG faker.factory:factory.py:106: Provider `faker.providers.python` does not feature localization. Specified locale `ru_RU` is not utilized for this provider.
2024-09-02 05:15:04.693 DEBUG faker.factory:factory.py:106: Provider `faker.providers.sbn` does not feature localization. Specified locale `ru_RU` is not utilized for this provider.
2024-09-02 05:15:04.693 DEBUG faker.factory:factory.py:76: Looking for locale `ru_RU` in provider `faker.providers.ssn`.
2024-09-02 05:15:04.695 DEBUG faker.factory:factory.py:95: Provider `faker.providers.ssn` has been localized to `ru_RU`.
2024-09-02 05:15:04.697 DEBUG faker.factory:factory.py:106: Provider `faker.providers.user_agent` does not feature localization. Specified locale `ru_RU` is not utilized for this provider.
2024-09-02 05:15:04.697 DEBUG tests_api.utils.logger:logger.py:23: > POST https://dev.agentum.pro/backend/api/users/profile {'method': 'POST', 'url': 'https://dev.agentum.pro/backend/api/users/profile', 'json': {'first_name': 'Адам', 'last_name': 'Сафонов', 'middle_name': 'Валентинович', 'employment_status': 'self-employed'}, 'headers': {'Content-type': 'application/json', 'Authorization': 'Bearer YwQogs0aDUHzYIgAIN88OBaTDiKesDmAdciD9vkz'}, 'params': None, 'files': None}
2024-09-02 05:15:04.699 DEBUG urllib3.connectionpool:connectionpool.py:1048: Starting new HTTPS connection (1): dev.agentum.pro:443
2024-09-02 05:15:05.054 DEBUG urllib3.connectionpool:connectionpool.py:546: https://dev.agentum.pro:443 "POST /backend/api/users/profile HTTP/1.1" 200 None
2024-09-02 05:15:05.055 DEBUG tests_api.utils.logger:logger.py:27: < 200 https://dev.agentum.pro/backend/api/users/profile {"data":{"id":48,"first_name":"\u0410\u0434\u0430\u043c","last_name":"\u0421\u0430\u0444\u043e\u043d\u043e\u0432","middle_name":"\u0412\u0430\u043b\u0435\u043d\u0442\u0438\u043d\u043e\u0432\u0438\u0447","phone":"79603848671","additional_phone":null,"employment_status":"self-employed","city":null,"avatar_url":"https:\/\/storage.yandexcloud.net\/pub-test\/avatar\/48\/01-09-2024-05-14-59.jpg","directions":[],"date_start":null,"about_myself":null,"email":null,"telegram":null,"whatsapp":null,"viber":null,"instagram":null}}
2024-09-02 05:15:05.055 INFO root:test_view_profile.py:45: {'data': {'id': 48, 'first_name': 'Адам', 'last_name': 'Сафонов', 'middle_name': 'Валентинович', 'phone': '79603848671', 'additional_phone': None, 'employment_status': 'self-employed', 'city': None, 'avatar_url': 'https://storage.yandexcloud.net/pub-test/avatar/48/01-09-2024-05-14-59.jpg', 'directions': [], 'date_start': None, 'about_myself': None, 'email': None, 'telegram': None, 'whatsapp': None, 'viber': None, 'instagram': None}}